certification/internal/shell/validate_operator_bundle.go

```diff
@@ -21,6 +22,11 @@ func (p ValidateOperatorBundlePolicy) Validate(bundle string) (bool, error) {
 	lines := strings.Split(string(stdouterr), "time=")
 
 	if strings.Contains(lines[len(lines)-1], "All validation tests have completed successfully") {
+		for _, line := range lines {
+			if strings.Contains(line, "level=warning") {
+				fmt.Println("time= ", line)
```

Comment:
@edreyer-1 can we log.Warn() here instead of fmt.Println()?

Reply:
I think this is all we need to merge!